src/common/Combobox/combobox.scss

```diff
@@ -9,6 +9,35 @@
   align-items: center;
   width: 100%;
 
+  &-input {
```

Comment:
don 't add some specific styles to common component styles, add them to styles where you need them